test/e2e_node/e2e_service.go

```diff
@@ -173,7 +173,7 @@ func (es *e2eService) startServer(cmd *healthCheckCommand) error {
 	go func() {
 		err := cmd.Run()
 		if err != nil {
-			cmdErrorChan <- fmt.Errorf("%s Failed with error \"%v\".  Command output:\n%v", cmd, err, *cmd.OutputBuffer)
+			cmdErrorChan <- fmt.Errorf("%s Failed with error \"%v\".  Command output:\n%s", cmd, err, cmd.OutputBuffer.String())
```

Comment:
I think with %s you need not call String() explicitly, cmd.OutputBuffer should be fine too. But doesn't matter much I think.